pkg/controller/iotconfig/infinispan_device_registry.go

```diff
@@ -58,6 +58,7 @@ func (r *ReconcileIoTConfig) reconcileInfinispanDeviceRegistryDeployment(config
 
 	install.ApplyDeploymentDefaults(deployment, "iot", deployment.Name)
 	deployment.Annotations[RegistryTypeAnnotation] = "infinispan"
+	deployment.Spec.Template.Spec.ServiceAccountName = "iot-device-registry"
```

Comment:
In order for this to work, we also do need to create, and set up the service account.

You have the YAML files in the templates folder, but we do require them as well in the CSV, for the operator manifest.

Then again, maybe we could let the IoT Config controller create it instead of using YAML files.

Reply:
I added resources to the csv for now. We can move this to the controller later on.